src/appl/gss-sample/gss-client.c

```diff
@@ -75,7 +75,7 @@ static gss_OID_desc gss_spnego_mechanism_oid_desc =
 {6, (void *)"\x2b\x06\x01\x05\x05\x02"};
 
 static void
-usage()
+usage(void)
```

Comment:
Does this specific definition actually cause a warning?  Although empty parens in a declaration makes the difference between a prototype and an old-style declaration, in a definition I think it's just a stylistic difference.

Reply:
That does not cause a warning, that was more of "well, since I'm in there ..." kind of thing.  Also, I did notice that the one build failed, but I am not quite sure why; it looks like it failed one of the unit tests but I'm not sure how these changed would have caused it.

Reply:
I will look into the build failure and see if it's related.